MassEffectModManagerCore/modmanager/localizations/rus.xaml

```diff
@@ -35,10 +35,10 @@
     <system:String x:Key="string_DownloadModMakermod">Скачать мод для ModMaker</system:String>
     <system:String x:Key="string_ASIModManager">ASI Mod Manager</system:String>
     <system:String x:Key="string_ASIManagerTooltip">Управление модами ASI. Только для продвинутых пользователей.</system:String>
-    <system:String x:Key="string_CustomDLCConflictDetector">Детектор Конфликтов Сustom DLC</system:String>
+    <system:String x:Key="string_CustomDLCConflictDetector">Сustom DLC Conflict Detector</system:String>
     <system:String x:Key="string_GUICompact">Генератор пакета совместимости для мода Interface Scaling/Controller (ME3)</system:String>
     <system:String x:Key="string_GUICompactTooltip">Проверяет моды ME3 на наличие экпортируемых элементов интерфейса, являющихся частью UI модов и генерирует патч-мод при необходимости</system:String>
-    <system:String x:Key="string_BatchModInstaller">Установщик Batch Mod</system:String>
+    <system:String x:Key="string_BatchModInstaller">Batch Mod Installer</system:String>
```

Comment:
In this instance, "Batch mod installer" means it is for installing a batch of mods. e.g. install 6 mods in a row. Eventually I plan to add this feature.